interface/src/app/greenelab/utils.js

```diff
@@ -0,0 +1,16 @@
+/**
+ * This module includes a few utility components that will be shared in other
+ * modules to avoid duplicate codes.
+ */
+
+angular.module('greenelab.utils', [])
+.constant('ApiBasePath', '/api/v0/')
```

Comment:
I think I understand @mhuyck's concern here. We have something called `greenelab/util.js` that may get abstracted and used in other servers in the future. Because this is specific to the ADAGE server, what we want is something like `adage/const.js`. Is this roughly it @mhuyck?